Content.Server/RandomMetadata/RandomMetadataComponent.cs

```diff
@@ -14,4 +14,7 @@ public sealed class RandomMetadataComponent : Component
 
     [DataField("nameSet", customTypeSerializer:typeof(PrototypeIdSerializer<DatasetPrototype>))]
     public string? NameSet;
+
+    [DataField("nameSet2", customTypeSerializer: typeof(PrototypeIdSerializer<DatasetPrototype>))]
+    public string? NameSet2;
```

Comment:
Doesn't sexextensions already have code handling this? Also you set the name once and then again if both are specified which is yucky.